frontend/src/sections/Navbar/Navbar.tsx

```diff
@@ -132,17 +132,19 @@ const Navbar = () => {
         <div className='flex h-12 w-full items-center justify-around md:h-20'>
           {/* logo  */}
           <div className='hidden items-center justify-center md:flex'>
-            <Image
-              src='/whatIsDjango/Logo.svg'
-              width={121}
-              height={42}
-              alt='logo'
-              className=' object-center'
-              style={{
-                maxWidth: '100%',
-                height: 'auto',
-              }}
-            />
+            <Link href='/' passHref>
+              <Image
+                src='/whatIsDjango/Logo.svg'
+                width={121}
+                height={42}
+                alt='logo'
+                className=' object-center'
+                style={{
+                  maxWidth: '100%',
+                  height: 'auto',
```

Comment:
Check if adding classnames for these styles will work or not

Reply:
I don't understand. Can you elaborate more on what you mean exactly?

Reply:
Sure. I was suggesting that instead of using style prop for adding styles, we can use tailwind classes for the same

Reply:
Thanks. I made the suggested changes.